### PR TITLE
Updated course index page with feedback link and updated copy.

### PR DESCRIPTION
**Pull request by dsjen**
@rocha @clintonb 

FYI: @shnayder @lamagnifica 

screen shot 2014-09-30 at 5 04 39 pm

### CODE REVIEW

File: acceptance_tests/test_course_index.py [hunk 1]
```diff
@@ -25,3 +28,7 @@ def test_course_list(self):
         # The element should link to the course landing page.
         href = element.attrs('href')[0]
         self.assertTrue(href.endswith('/courses/{}/'.format(course_id)))
+
+        # check that we have an email
+        element = self.page.q(css='div[class=help-msg] a[class=feedback-email]')
+        self.assertEqual(element.text[0], DASHBOARD_FEEDBACK_EMAIL)
```
Comment (clintonb) [inline acceptance_tests/test_course_index.py]: Minor: Pull this into its own method, `assertValidFeedbackLink(css_selector)` and use in `FooterMixin._test_footer`.

**Comment by clintonb** [comment]
:+1:

**Comment by lamagnifica** [comment]
:+1:

File: acceptance_tests/test_course_index.py (hunk 1 above)
Comment (dsjen) [inline acceptance_tests/test_course_index.py]: I moved a few of the shared assertions into `AssertMixin`, created a new `assertValidFeedbackLink` method used in `_test_footer` in addition to `test_course_list`.

**Comment by rocha** [comment]
:rocket:

**Pull request merged**